python/src/sentencepiece/__init__.py

```
@@ -4,7 +4,14 @@
 # Do not make changes to this file unless you know what you are doing - modify
 # the SWIG interface file instead.
 
+import re
+import csv
+import sys
+import os
+from io import StringIO
+from io import BytesIO
 from sys import version_info as _swig_python_version_info
```

Ruff

`sys.version_info` imported but unused

(F401)